slack.json

```diff
@@ -0,0 +1,19 @@
+{
+  "runtime": "deno",
+  "hooks": {
+    "build": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-write --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts",
+    "get-manifest": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts --manifest",
+    "start": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://deno.land/x/deno_slack_runtime@0.0.3/mod.ts"
+  },
+  "scripts": {
```

Comment:
Closing the loop here, @sopokuagyemang can we remove `scripts` for the time being until it can be properly supported by hooks lib?

Reply:
My understanding was we would close this PR and instead move forward with https://github.com/slackapi/deno-reverse-string/pull/22